package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@luckbox/eslint-rules",
-  "version": "2.2.5",
+  "version": "3.0.0",
```

Comment:
I can say yes since we have incompatible changes, we are now using peerDependencies and we have split the project to ts and non ts (be) rules so microservices that are not migrated to typescript can use the new version of the eslint.